robot_calibration/src/finders/checkerboard_finder.cpp

```diff
@@ -111,15 +117,19 @@ bool CheckerboardFinder::waitForCloud()
 
 bool CheckerboardFinder::find(robot_calibration_msgs::CalibrationData * msg)
 {
+  const int count = 50;
+  ROS_INFO_STREAM("Trying to detect checkboard for max " << count << " times.");
   // Try up to 50 frames
-  for (int i = 0; i < 50; ++i)
+  for (int i = 0; i < count; ++i)
   {
     // temporary copy of msg, so we throw away all changes if findInternal() returns false
     robot_calibration_msgs::CalibrationData tmp_msg(*msg);
     if (findInternal(&tmp_msg))
     {
       *msg = tmp_msg;
       return true;
+    } else {
+      ROS_INFO_STREAM("No checkboard found in iteration: " << i);
```

Comment:
this will be very spammy - I'd suggest making this a ROS_DEBUG_NAMED so it's easy to turn on, but is not printed by default.